packages/irrlicht.rb

```diff
@@ -13,23 +13,41 @@ class Irrlicht < Package
   depends_on 'libxxf86vm'
   depends_on 'libjpeg'
   depends_on 'libpng'
+  depends_on 'dos2unix' => :build
+  
+  def self.patch
+    system "curl -#LO https://dev.gentoo.org/~mgorny/dist/irrlicht-1.8.4-patchset.tar.bz2"
+    @sha256sums = <<~EOF
+      03b6a5a8a98191f8efe6f7ccd5c957bbf2c4bf15ac6cdc987655792c8baff454  irrlicht-1.8.4-patchset.tar.bz2
+    EOF
+    IO.write("sha256sums", @sha256sums)
+    system "sha256sum -c sha256sums"
+    system "sed -i 's:\.\./\.\./media:../media:g' $(grep -rl '\.\./\.\./media' examples)"
+    system "sed -i 's/\r$//' source/Irrlicht/COSOperator.cpp"
+    system "dos2unix include/IrrCompileConfig.h"
+    system "tar xvf irrlicht-1.8.4-patchset.tar.bz2"
+    system "for patch in irrlicht-1.8.4-patchset/*; do patch -p 1 -i ${patch}; done"
+    system "sed -i 's:-I/usr/include/freetype2/:-I#{CREW_PREFIX}/include/freetype2:g' $(find -name Makefile)"
+    system "sed -i 's:-I/usr/X11R6/include:-I#{CREW_PREFIX}/X11R6/include:g' $(find -name Makefile)"
+    system "sed -i 's:-L/usr/X11R6/lib$(LIBSELECT):-L/usr/local/X11R6/lib$(LIBSELECT):g' $(find -name Makefile)"
+    system "sed -i 's:-I/usr/include:-I#{CREW_PREFIX}/include:g' $(find -name Makefile)"
+  end
   
   def self.build
     Dir.chdir 'source/Irrlicht' do
-      system 'make'
-      system 'make clean'
-      system 'make sharedlib'
+      system 'make sharedlib staticlib'
     end
   end
 
   def self.install
-    FileUtils.mkdir_p CREW_DEST_LIB_PREFIX
+    FileUtils.mkdir_p "#{CREW_DEST_LIB_PREFIX}"
```

Comment:
This line doesn't need changed. :)

Reply:
It was an inconsistency that was annoying me. I also find it reduces my readability.

Reply:
I thought we just made this change on a previous commit?

Reply:
Would you like me to change it back?

Reply:
No need but it would be less characters to wade through when checking.